eng/versioning/version_set_dev.py

```diff
@@ -27,28 +38,38 @@ def is_in_service(sdk_path, setup_py_location, service_name):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Increments version for a given package name based on the released version')
-    parser.add_argument('--sdk-path', default=DEFAULT_SDK_PATH, help='path to the sdk folder')
-    parser.add_argument('--service-name', required=True, help='name of the service for which to set the dev build id (e.g. keyvault)')
-    parser.add_argument('--build-id', required=True, help='id of the build (generally of the form YYYYMMDD.r) dot characters(.) will be removed')
 
-    args = parser.parse_args()
+    parser.add_argument(
+        "glob_string",
+        nargs="?",
+        help=(
+            "A comma separated list of glob strings that will target the top level directories that contain packages."
+            'Examples: All = "azure-*", Single = "azure-keyvault", Targeted Multiple = "azure-keyvault,azure-mgmt-resource"'
+        ),
+    )
 
-    service_name = args.service_name.replace('_', '-')
-    build_id = format_build_id(args.build_id)
-    sdk_path = args.sdk_path
+    parser.add_argument('--service', help='name of the service for which to set the dev build id (e.g. keyvault)')
+    parser.add_argument("-b", '--build-id', required=True, help='id of the build (generally of the form YYYYMMDD.r) dot characters(.) will be removed')
+
+    args = parser.parse_args()
 
-    packages = get_packages(sdk_path)
-    target_packages = [pkg for pkg in packages if is_in_service(sdk_path, pkg[0], service_name)]
+    if args.service:
```

Comment:
Looks like this logic is dulicated in all the script should we move it into the get_packages function or another shared helper?

Reply:
Thanks. I will make this change.